fleet/lib/fleet-conf/overlays/ls/gitrepo-rancher.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: fleet.cattle.io/v1alpha1
+kind: GitRepo
+metadata:
+  name: rancher
+  namespace: fleet-local
+spec:
+  repo: https://github.com/lsst-it/k8s-cookbook
+  branch: master
```

Comment:
Clusters in `ls` should be using the `ls_production` branch.

Reply:
changed to ls_production 👍🏻  my bad